packages/types/.changesets/remove-type-restrictions-on-span-params.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "remove"
+---
+
+Remove type restrictions on Span params
```

Comment:
I'm also happy with not having this changelog at all. I don't think the type of the change should be "remove", in that we're not deleting any functionality from the application.

```suggestion
type: "change"
---

Update the type definition of our `setParams` and `params` span interfaces so that nested objects and arrays can be sent inside the params object.
```